### ISSUE TITLE
Just FYI, Pro Status does not update between apps unless you log out then in

### ISSUE THREAD

**Issue by chriseich21** [k0shk0sh/FastHub#994]
**FastHub Version: 4.2.0**  
**Android Version: 7.1.1 (SDK: 25)**  
**Device Information:**  
- **Manufacturer:** OnePlus  
- **Brand:** OnePlus  
- **Model:** OnePlus3T  
---

This isn't really a bug, but I would add a way to restore purchases within the actual app, as not everyone may think of logging out and back in. It should not be an issue if they download it new, but my old login on my tablet did not update after I paid on my phone and attempted to charge me again.

Thanks for the great app too!

**Comment by k0shk0sh** [comment]
is your primary  email in both phones are the same? because I'm checking in Everytime you open the app to restore your purchases. if not then I can't do anything about it and its Google play problem

**Comment by k0shk0sh** [comment]
this is the splash activity that u seen when u open the app and its responsible for these stuff 

https://github.com/k0shk0sh/FastHub/blob/master/app/src/main/java/com/fastaccess/ui/modules/main/donation/CheckPurchaseActivity.kt

**Comment by chriseich21** [comment]
Yea it is the same Github account (I only have 1) and I used Oauth through
Chrome. It did not update when I closed the app in background and restarted
FastHub, but when I logged out of my Github account and back in it updated.
Like I said it's not a major issue, was mostly a suggestion!

**Comment by k0shk0sh** [comment]
it is not tied to your GitHub account but to your Gmail account!

**Comment by chriseich21** [comment]
Sorry, yes the Gmail account is the same and primary on both.

I purchased Pro on phone in the morning and everything worked fine. That night I opened the application on my tablet (which was previously powered off, it is Samsung Galaxy Tab A with S Pen, android 7.0 I believe) and tried to change the theme, it popped up with the $1 payment. I exited the application, reopened it and attempted to change the theme again, it popped up with the $1 charge.

This time I signed out of my Github account within the FastHub application and signed back in. Once I did that, it allowed me to change the theme for free. Sorry, that is the best summary I can give and I cannot reproduce it as once I logged in/out of Github account within FastHub and it synced.
